### DISCUSSION TITLE
Some Sensors has wrong units

### DISCUSSION THREAD

**Discussion by Thelch** [sfstar/hass-victron#95] [Q&A]
Hello i see in my HA that i got some wrong units from the victron solution:

the pvinverter often shows W and WH as kWh or kW.

How can i fix this?
![image](https://github.com/sfstar/hass-victron/assets/17222597/47a25cf7-5786-4385-8983-10058276b770)

Thanks for help :-)

**Comment by randyoo** [comment]
I think you may have discovered a bug related to confusion over the meaning of “.” and “,”.

In the US, the “.” indicates end of whole number, start of fractions. If you look at those numbers through that lens, then the units are correct, right?

Maybe @sfstar can jump in to tell if this is a bug with this integration or in HA itself. Meanwhile, you could try switching your settings in HA. If you choose Settings, General, then choose US unit system, or change country/language to US/English, problem is gone?

**Reply by Thelch** [reply]
Hello randyoo,

i tried your solution but error is still the same:
![image](https://github.com/sfstar/hass-victron/assets/17222597/be70deb0-7f3c-40e9-a8ac-cf2d731c7160)
![image](https://github.com/sfstar/hass-victron/assets/17222597/43c7eb82-373b-4f61-a970-061c1c2c1101)
![image](https://github.com/sfstar/hass-victron/assets/17222597/c891898d-8dcb-4fbb-a2d5-ebdc3a14d16c)

If i cange the unit to W the number will be multiplied by 1000 and locks more terribel
![image](https://github.com/sfstar/hass-victron/assets/17222597/4e4207f5-7279-4b01-a577-b766c5389674)

**Reply by randyoo** [reply]
Wow, that is really ridiculous!

I'm afraid I don't have any more ideas for you. But I don't think you're the only user in Europe. My only suggestion is to open an issue, hopefully it will get the attention of @sfstar and they will have a look...
